lib/util.js

```diff
@@ -19,22 +20,42 @@ const requestDefaults = {
         'User-Agent': USER_AGENT,
     },
 };
-let pooledRequest = request.defaults(requestDefaults);
+
+// Exponential wait: 800ms, 1600ms, 3200ms
+const MAX_RETRIES = 3;
+const RETRY_START_MS = 800;
 
 // Wrap request results in a promise
-function rp (options) {
-    return new Promise((resolve, reject) => {
-        pooledRequest(options, (err, res, body) => {
-            if (err) return reject(err);
-            resolve(res);
-        });
+let rp = promisify(request.defaults(requestDefaults));
```

Comment:
We generally try to avoid promisifying since it makes the API harder to understand. On top of this, there are modules that already do this: https://github.com/request/request#promises--asyncawait

Reply:
PS: This should be a const